docs/src/api/class-cdpsession.md

```diff
@@ -63,17 +77,22 @@ send messages.
 
 ## async method: CDPSession.send
 * since: v1.8
-* langs: js, python, csharp
+* langs: js, python, csharp, java
```

Comment:
Here and in the other places you can simply drop the  `langs` filter as the method is now available in all languages.